### PR TITLE
Phase3

### PR DESCRIPTION
**Pull request by pineaultratwit**
Save form data to messages collection in firestore. Anonomous auth controlling requests.

@alpinealex

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: components/ContactForm.tsx [hunk 1]
```diff
@@ -12,11 +14,33 @@ export interface IContactFormData {
 
 const ContactForm = () => {
   const { register, handleSubmit } = useForm<IContactFormData>();
+  const loggedIn = useRef(false);
```
Comment (microgreensgrowapp) [inline components/ContactForm.tsx:17]: Curious why you used `useRef` instead of `useState`?

**Review by microgreensgrowapp — COMMENTED** [review]
Also, what is the reason for merging in your own PRs instead of waiting for me to peer review and approve the merge first?

File: components/ContactForm.tsx (hunk 1 above)
Comment (pineaultratwit) [reply components/ContactForm.tsx:17]: nothing is changing on the ui when loggedIn is updated

Comment (alpinealex) [reply components/ContactForm.tsx:17]: I would argue the `loggedIn` variable should be used to enable/disable the submit button state via the `disabled` prop.

**Pull request merged**